### DISCUSSION TITLE
Question about fan speed of ducted ceiling unit (SEZ-M35DA).

### DISCUSSION THREAD

**Discussion by MortuuS** [echavet/MitsubishiCN105ESPHome#142] [Q&A]
Hi!
First of all thanks for this plugin. Finally I no longer have issues with the state being stuck on idle or it not responding to changes as with the version of Geoff Davis.

But with the extra information provided by this plugin I'm not sure about the fan speed of my unit as indicated by the stage sensor.
In my yaml I have these fan speeds 
`fan_mode: ['AUTO', 'LOW', 'MEDIUM', 'HIGH']`
When in select HIGH in HASS the stage sensor shows 'moderate' as the speed. Is this correct? Or does my unit have a 'higher' fan speed?  Sometimes I feel like my unit might be under performing could this be the cause? Or should I be looking into the static pressure settings? However i'm unable to double check the state on anything official Mitsubishi since I only have a ESP32 connected to this unit.

Thanks :)

**Comment by echavet** [comment]
> Hi!
> 
> First of all thanks for this plugin. Finally I no longer have issues with the state being stuck on idle or it not responding to changes as with the version of Geoff Davis.
> 
> 
> 
> But with the extra information provided by this plugin I'm not sure about the fan speed of my unit as indicated by the stage sensor.
> 
> In my yaml I have these fan speeds 
> 
> `fan_mode: ['AUTO', 'LOW', 'MEDIUM', 'HIGH']`
> 
> When in select HIGH in HASS the stage sensor shows 'moderate' as the speed. Is this correct? Or does my unit have a 'higher' fan speed?  Sometimes I feel like my unit might be under performing could this be the cause? Or should I be looking into the static pressure settings? However i'm unable to double check the state on anything official Mitsubishi since I only have a ESP32 connected to this unit.
> 
> 
> 
> Thanks :)

Hey @MortuuS 
Thanks for your feedback 
The stage sensor is about the external unit, not the inside unit 
Fan mode is for inside unit
You have no control on the outside fan speed as this is internal stuff for managing compressor work
